docs/azure_jumpstart_hcibox/cloud_deployment/_index.md

```diff
@@ -60,7 +68,7 @@ If you already have [Microsoft Defender for Cloud](https://learn.microsoft.com/a
 
   ![Screenshot showing connecting to the VM using JIT](./jit_rdp_connect.png)
 
-#### The Logon scripts
+### The Logon scripts
 
 - Once you log into the _HCIBox-Client_ VM, a PowerShell script will open and start running. This script will take anywhere between 1-2 hours to finish, and once completed, the script window will close automatically. At this point, the infrastructure deployment is complete.
```

Comment:
The deployment takes a bit more than 4 hours when also updating the cluster to the latest update. This is something we added in recent months, and adds around 1 hour to the deployment time. I would change the indication to 2-3 hours, plus 1 hour if any HCI updates is pending.